tests/integration/cqlengine/connections/test_connection.py

```diff
@@ -127,3 +130,74 @@ def test_connection_session_switch(self):
         connection.set_session(self.session2)
         self.assertEqual(1, TestConnectModel.objects.count())
         self.assertEqual(TestConnectModel.objects.first(), TCM2)
+
+
+class LegacyConnectionsModel(Model):
```

Comment:
Not sure the best place for these tests. We have a `test_connection.py` and `test_connections.py`, and I'm not really sure the difference between them.